src/panoptes/pocs/focuser/astromechanics.py

```diff
@@ -44,7 +44,7 @@ def position(self):
         """
         response = ''
         try:
-            response = int(self._send_command("P").rstrip("#"))
+            response = int(self._send_command("P").rstrip("#")) - int(self._calibration_position)
```

Comment:
Maybe worth a comment to explain why the calibration position is being subtracted here.

Reply:
👍

Reply:
```suggestion
            response = int(self._send_command("P").rstrip("#")) - int(self._zero_position)
```